simulation.go

```diff
@@ -264,7 +264,7 @@ func (s *SimulationBFTree) CreateRoster(sc *SimulationConfig, addresses []string
 		if sc.PrivateKeys[entities[0].Address].Equal(
```

Comment:
I don't really understand this line actually... Was it just the "check" to make sure we were not using the same key for all nodes ? 
Why do you think it is not needed anymore ?